index.html

```diff
@@ -16,5 +16,166 @@
 <body>
     <!-- load javascript after loading all html content -->
     <script src="script/script.js"></script>
+    <br> </br>
+    <br> </br>
+    <h1> This is Jayan's Website</h1>
+    <br> </br>  <!-- This creates a space between the two lines of text -->
+    
+    <nav class="navbar navbar-expand-lg fixed-top navbarScroll">
+        <div class="container">
+            <a class="navbar-brand" href="#">Jayan</a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav ms-auto">
+                    <li class="nav-item active">
+                        <a class="nav-link" href="#home">Home</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#about">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#services">Services</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#portfolio">Portfolio</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#contact">Contact</a>
+                    </li>
+                </ul>
+                
+            </div>
+        </div>
+    </nav>
+    <section class="bgimage" id="home">
+        <div class="container-fluid">
+            <div class="row">
+            <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12 hero-text">
+                <h2 class="hero_title">Hi, it's me Jayan</h2>
+                <p class="hero_desc">I am 14 years old and I enjoy math and sports</p>
+            </div>
+            </div>
+        </div>
+    </section>
+    <section id="about">
+        <div class="container mt-4 pt-4">
+            <h1 class="text-center">About Me</h1>
+            <div class="row mt-4">
+                <div class="col-lg-4">
+                    <img src="images/about.jpeg" class= "imageAboutPage" alt="">
```

Comment:
This image doesn't exist, so there is a big blank:

<img width="1366" alt="Screen Shot 2022-06-24 at 5 10 22 PM" src="https://user-images.githubusercontent.com/6845676/175750242-571368ba-b198-40e4-8f96-a2b6904e093a.png">

When designing a website, I often don't have the images yet. To get around this, you can use placeholder image sites, which make it really easy to get images of a particular size:


- https://place-puppy.com/
- http://www.placecage.com/
- https://baconmockup.com/